react/components/ResourceList/index.js

```diff
@@ -200,6 +234,50 @@ class ResourceList extends PureComponent {
                 </span>
               </Button>
             )}
+            {isExtraActionsVisible && (
+              <div
+                id="toggleExtraActionsBtn"
+                ref={this.extraActionsBtnRef}
+                className="relative">
+                <Button
+                  variation="tertiary"
+                  size="small"
+                  // eslint-disable-next-line react/jsx-handler-names
+                  onClick={this.toggleExtraActionsSelector}
```

Comment:
muda pra `this.handle...Click` (substitui `...` pelo nome do botao), e o handler que chame esse toggle